tests/e2e/tests/dashboard/fortio-rules.yaml

```diff
@@ -9,39 +9,39 @@ spec:
   selector:
     app: echosrv
 ---
-apiVersion: extensions/v1beta1
-kind: Ingress
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
 metadata:
-  annotations:
-    kubernetes.io/ingress.class: istio
-  name: istio-ingress
+  name: fortio-gateway
 spec:
-  rules:
-    - http:
-        paths:
-          - path: /fortio/.*
-            backend:
-              serviceName: echosrv
-              servicePort: http-echo
-          - path: /tcp/.*
-            backend:
-              serviceName: tcpechosrv
-              servicePort: tcp-echo
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - echosrv
```

Comment:
wont work unless your client is literally supplying echosrv as the HTTP host.

Reply:
👍 should I just remove the `hosts` section altogether?  I'm stuck just trying to get the tests to run, so I haven't had time to debug the setup.